### ISSUE TITLE
Support for the Ke parameter in mtl file

### ISSUE THREAD

**Issue by Lesliehdezg** [gpvigano/AsImpL#17]
In the material library file (mtl) the parameter Ke (emissive 
coefficient) represents the amount of light emitted by a material. In 
Unity could be somehow mapped to the Emission value of Material.

**Comment by gpvigano** [comment]
I just added the support for emission color to the last version of AsImpL in [this commit](48b63c2cf85da0fe0403bda53bd397cf694bfe60). 

A new field `emissiveColor` is defined in `MaterialData`.
The `Ke` parameter is read from mtl file (`LoaderObj`) and imported to Unity materials (`ObjectBuilder`).
The `DiffBump` material in `objtest.mtl` now has a new `Ke` parameter to test emissive color.

Notice that in addition to setting the emission color of the material I had to enable the `_EMISSION` keyword.
An hint about implementing new features: right click in the Inspector tab label and select `Debug` from the context menu (to switch back to normal view just right click again and select `Normal`).
In this way you have a full overview on the real component parameters.
I found this way very useful to discover Unity materials internal configuration.
